datasource/local/src/main/java/com/egdroid/DataSource/local/MovieLocal.java

```diff
@@ -0,0 +1,53 @@
+package com.egdroid.DataSource.local;
+
+import androidx.room.PrimaryKey;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class MovieLocal { // from Room
```

Comment:
Are we sure this is the local model? 😄 I guess we need to change it to MovieApiResponse for example
```suggestion
public class MovieApiResponse { // for retrofit
```

Reply:
Room models should have `@Entity` on the class level